release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1050,6 +1050,16 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-monitoring-bug-fixes"]
 ==== Monitoring
 
+* Before this update, the `prometheus-adapter` pods might flood the Prometheus HTTP servers with too many requests when users requested pod metrics across all namespaces. The issue also became visible when the Vertical Pod Autoscaler was deployed. In the worst cases, `prometheus-adapter` pods would hit their maximum TCP connection request limit, and liveness probes might fail, which led to Prometheus pod restarts. With this update, the requests from a `prometheus-adapter` pod to the Prometheus service are limited to 100 simultaneous requests, thereby fixing the issue. link:https://issues.redhat.com/browse/OCPBUGS-20250[OCPBUGS-20250]
+
+* Before this update, node-exporter could consume large amounts of CPU resources in some configurations. This update limits the number of CPUs that can be used simultaneously by node-exporter. The number of CPUs is now set to `4` or to the number of CPUs available on the host, whichever is smaller. You can override this limit by using the `maxProcs` parameter in the config map settings for node-exporter. See xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.html#nodeexporterconfig[NodeExporterConfig] for more information. link:https://issues.redhat.com/browse/OCPBUGS-13153[OCPBUGS-13153]
```

Comment:
```suggestion
* Before this update, the node-exporter could consume large amounts of CPU resources in some configurations. This update limits the number of CPUs that can be used simultaneously by node-exporter. The number of CPUs is now set to `4` or to the number of CPUs available on the host, whichever is smaller. You can override this limit by using the `maxProcs` parameter in the config map settings for the node-exporter. See xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.html#nodeexporterconfig[NodeExporterConfig] for more information. link:https://issues.redhat.com/browse/OCPBUGS-13153[OCPBUGS-13153]
```

Reply:
Fixed, and thank you for catching the ** issue!